src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AdHocTestkeys } from 'common/configs/adhoc-test-keys';
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { generateUID } from 'common/uid-generator';
+import { AdhocIssuesTestView } from 'DetailsView/components/adhoc-issues-test-view';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import * as React from 'react';
+
+export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
+    key: AdHocTestkeys.NeedsReview,
+    testMode: TestMode.Adhoc,
+    getTestView: props => <AdhocIssuesTestView {...props} />,
+    getStoreData: data => data.adhoc.needsReview,
+    enableTest: (data, _) => (data.enabled = true),
```

Comment:
Question as I'm getting caught up on this PR: what's the significance of the empty parameter for `enableTest`?

Reply:
it's the ToggleActionPayload

Reply:
Ok talked to Wahaj, and the ToggleActionPayload is not being used here for enable test (hence the `_`) so it's ok to remove it and just have `enableTest: (data) ="> (data.enabled = true)`

Reply:
Great, I'll make the change